core/src/main/kotlin/com/justai/jaicf/context/ProcessContext.kt

```diff
@@ -2,6 +2,7 @@ package com.justai.jaicf.context
 
 import com.justai.jaicf.activator.ActivationContext
 import com.justai.jaicf.api.BotRequest
+import com.justai.jaicf.logging.LoggingContext
```

Comment:
`ProcessContext` and `LogginContext` have lots of common properties, so in my opinion, we should better think what properties related to `ProcessContext` and what to `LoggingContext` and reuse them.
Maybe it'd better add `ProcessContext` property to `LoggingContext` and don't repeat other common properties.